test-suite/output/PrintAssumptions.out

```diff
@@ -1,27 +1,32 @@
 Axioms:
-foo : nat
+foo
+ : nat
```

Comment:
I don't know why it forces a newline even when it's short like this.

Reply:
lack of boxes?